<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>AboutDlgBase</class>
  <widget class="QDialog" name="AboutDlgBase" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>200</height>
      </rect>
    </property>
    <property name="sizePolicy" >
      <sizepolicy>
        <hsizetype>0</hsizetype>
        <vsizetype>0</vsizetype>
      </sizepolicy>
    </property>
    <property name="minimumSize" >
      <size>
        <width>400</width>
        <height>200</height>
      </size>
    </property>
    <property name="maximumSize" >
      <size>
        <width>400</width>
        <height>200</height>
      </size>
    </property>
    <property name="windowTitle" >
      <string>About Unizone</string>
    </property>
    <property name="sizeGripEnabled" >
      <bool>false</bool>
    </property>
    <widget class="QWidget" name="Layout5" >
      <property name="geometry" >
        <rect>
          <x>0</x>
          <y>0</y>
          <width>397</width>
          <height>197</height>
        </rect>
      </property>
      <layout class="QVBoxLayout" >
        <property name="margin" >
          <number>0</number>
        </property>
        <property name="spacing" >
          <number>6</number>
        </property>
        <item>
          <layout class="QVBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <widget class="QLabel" name="titleLabel" >
                <property name="font" >
                  <font>
                    <pointsize>16</pointsize>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="text" >
                  <string/>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap" >
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="versionLabel" >
                <property name="font" >
                  <font>
                    <pointsize>12</pointsize>
                  </font>
                </property>
                <property name="text" >
                  <string/>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="TextLabel2" >
                <property name="text" >
                  <string>Written by Mika T. Lindqvist</string>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="TextLabel2_2" >
                <property name="text" >
                  <string>Based upon the Windows MUSCLE client Ozone</string>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="TextLabel2_3" >
                <property name="text" >
                  <string>Which was written by Vitaliy Mikitchenko</string>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="TextLabel2_3_2" >
                <property name="text" >
                  <string/>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QHBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <spacer name="Spacer1" >
                <property name="sizeHint" >
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item>
              <widget class="QPushButton" name="buttonOk" >
                <property name="windowTitle" >
                  <string/>
                </property>
                <property name="text" >
                  <string>&amp;OK</string>
                </property>
                <property name="autoDefault" >
                  <bool>true</bool>
                </property>
                <property name="default" >
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item>
              <spacer name="Spacer2" >
                <property name="sizeHint" >
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
  </widget>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
